gallery_dl/extractor/sankaku.py

```diff
@@ -150,6 +152,8 @@ class SankakuTagExtractor(SankakuExtractor):
     def __init__(self, match):
         SankakuExtractor.__init__(self)
         self.tags = text.unquote(match.group(1).replace("+", " "))
+        self.start_page = 1 if match.group(3) is None else int(text.unquote(match.group(3)))
```

Comment:
Use `util.safe_int(..., 1)`.
You don't need to check if this particular group is `None`, safe_int() already handles that.

Also don't unquote something that should be a number.